config/config/src/main/java/io/helidon/config/BuilderImpl.java

```diff
@@ -605,6 +584,7 @@ private EmptyConfigHolder() {
                 .disableSystemPropertiesSource()
                 .disableParserServices()
                 .disableFilterServices()
+                .changesExecutor(command -> {})
```

Comment:
Maybe we could add some `disableMutableSources` or something similar in the future